translations/ai_rules.tex/pl.po

```diff
@@ -227,26 +227,14 @@ msgstr ""
 
 #. type: multicols
 #: sections/ai_rules.tex:74
-#, fuzzy, no-wrap
-#| msgid ""
-#| "Enemy Ground \\includesvg[height=10px]{\\svgs/unit_ground.svg} and Flying \\includesvg[height=10px]{\\svgs/unit_flying.svg} Units prioritize attacking Units of the same tier.\n"
-#| "If this is impossible, they attack the closest Unit, prioritizing the lowest tier one.\\par\n"
-#| "Ranged \\includesvg[height=10px]{\\svgs/unit_ranged.svg} Units prioritize attacking other Ranged \\includesvg[height=10px]{\\svgs/unit_ranged.svg} Units of the same tier, then lower tier, and finally higher tier.\n"
-#| "If there are no Ranged \\includesvg[height=10px]{\\svgs/unit_ranged.svg} Units for them to target, they prioritize Ground \\includesvg[height=10px]{\\svgs/unit_ground.svg} and Flying \\includesvg[height=10px]{\\svgs/unit_flying.svg} Units in the same tier order.\n"
-#| "If there's more than one valid target, they attack the closest one.\\par\n"
-#| "If there's ever a tie between equally valid targets for any Units, the player chooses which Unit is attacked.\n"
-#| "Enemy units cannot \\hyperlink{Defend}{Defend} unless instructed to.\n"
-#| "\n"
+#, no-wrap
 msgid ""
 "Enemy Ground \\includesvg[height=10px]{\\svgs/unit_ground.svg} and Flying \\includesvg[height=10px]{\\svgs/unit_flying.svg} Units prioritize attacking Units of the \\textbf{same} tier.\n"
 "If this is impossible, they attack the Unit of a lower tier (in tier \\textbf{descending} order, down to bronze), and if that is also impossible, they attack the Unit of a higher tier (in tier \\textbf{ascending} order).\\par\n"
 "\n"
 msgstr ""
 "Wrogie Jednostki Naziemne \\includesvg[height=10px]{\\svgs/unit_ground.svg} oraz Latające \\includesvg[height=10px]{\\svgs/unit_flying.svg} atakują w pierwszej kolejności Jednostki \\textbf{tego samego} poziomu.\n"
-"Jeśli jest to niemożliwe, atakują Jednostkę niższego poziomu (w malejącej kolejności aż do brązowego), a jeśli i to jest niemożliwe, Jednostkę wyższego poziomu (w rosnącej kolejności poziomów).\\par\n"
-"Jednostki Strzelające \\includesvg[height=10px]{\\svgs/unit_ranged.svg} atakują najpierw inne Jednostki Strzelające \\includesvg[height=10px]{\\svgs/unit_ranged.svg} tego samego poziomu, następnie niższego poziomu, a na końcu wyższego w takiej samej kolejności jak napisano powyżej.\n"
-"Jeśli nie ma Jednostek Strzelających, które mogłyby być celem Ataku, w pierwszej kolejności atakują Jednostki Lądowe \\includesvg[height=10px]{\\svgs/unit_ground.svg} oraz Latające \\includesvg[height=10px]{\\svgs/unit_flying.svg} w tej samej kolejności poziomów.\\par\n"
-"\n"
+"Jeśli jest to niemożliwe, atakują Jednostkę niższego poziomu (w \\textbf{malejącej} kolejności aż do brązowego), a jeśli i to jest niemożliwe, Jednostkę wyższego poziomu (w \\textbf{rosnącej} kolejności poziomów).\\par\n"
```

Comment:
to tylko sugestia, ale zastanawiam się, czy skrót nie będzie za duży

```suggestion
"Jeśli jest to niemożliwe, atakują Jednostkę niższego poziomu (w \\textbf{malejącej} kolejności aż 
do brązowego), a jeśli i to jest niemożliwe, atakują Jednostkę wyższego poziomu (w \\textbf{rosnącej} kolejności poziomów).\\par\n"
```

Reply:
mimo, że to zdanie wielokrotnie złożone - myślę, że raz użyty "czasownik: atakują" jest wystarczający i nie trzeba go ponownie stosować... dzięki takiemu zabiegowi też zmieściliśmy zasady AI na dwóch stronach. Ja bym tą zmianę jednak odrzucił...

Reply:
ok